### DISCUSSION TITLE
Difference with Angulartics2

### DISCUSSION THREAD

**Discussion by Swordfish815** [Arnaud73/ngx-matomo#63] [Q&A] [answered]
Hello Arnaud,

First I'd like to thank you for your work on this wrapper, it makes integrating with Matomo really easy.
You mentioned somewhere that you were indirectly inspired by angulartics2.
Could you please tell me what are the differences between ngx-matomo and angulartics2 (for angular).

Best regards,

**Comment by Arnaud73** [comment, marked as answer]
I've used Angulartics2 a few years ago while needing to track actions in my app to a Piwik instance.

To my understanding, Angulartics2 is a vendor agnostic solution. It aims to provide a single API to enable tracking with any solution.

ngx Matomo has been built with just a few objectives:
* make it as small as possible
* offer a simple interface to most of Matomo features

I'd say you shall not use ngx Matomo if you need to track the use with 2 or more web analytics solutions or if you may need to switch to a different vendor in a foreseeable future. Else, if Matomo is your solution of choice, ngx Matomo is a solution dedicated to enable you to get the most of it.
